media-gfx/nomacs/nomacs-3.17.2282.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PLUGIN_PKG="${PN}-plugins-$(ver_cut 1-2)"
+inherit cmake xdg-utils
+
+DESCRIPTION="Qt-based image viewer"
+HOMEPAGE="https://nomacs.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+plugins? ( https://github.com/${PN}/${PN}-plugins/archive/$(ver_cut 1-2).tar.gz -> ${PLUGIN_PKG}.tar.gz )"
```

Comment:
How is that supposed to work without nomacs-plugins-3.17 having been tagged? What exactly did you manifest there?

```
>>> Downloading 'https://github.com/nomacs/nomacs-plugins/archive/3.17.tar.gz'
--2023-09-03 11:51:29--  https://github.com/nomacs/nomacs-plugins/archive/3.17.tar.gz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/nomacs/nomacs-plugins/tar.gz/3.17 [following]
--2023-09-03 11:51:29--  https://codeload.github.com/nomacs/nomacs-plugins/tar.gz/3.17
Resolving codeload.github.com (codeload.github.com)... 140.82.121.10
Connecting to codeload.github.com (codeload.github.com)|140.82.121.10|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-09-03 11:51:30 ERROR 404: Not Found.

!!! Couldn't download 'nomacs-plugins-3.17.tar.gz'. Aborting.
```

Reply:
Yeah, I couldn't find it/its not tagged